services/QuillLMS/client/app/bundles/Evidence/components/studentView/container.tsx

```diff
@@ -59,7 +59,7 @@ const MINIMUM_STUDENT_HIGHLIGHT_COUNT = 2
 
 export const StudentViewContainer = ({ dispatch, session, isTurk, location, activities, handleFinishActivity, user, }: StudentViewContainerProps) => {
   const activityCompletionCount: number = parseInt(getParameterByName('activities', window.location.href, '0'));
```

Comment:
I think something funny is going on here - '0' is being submitted as a third argument to `getParameterByName` but that function only takes two arguments. My guess is that it's supposed to be a default, and if we rewrite this line as I have it below it'll clean up some of the conditional logic below.

` const activityCompletionCount: number = parseInt(getParameterByName('activities', window.location.href)) || 0;`

Reply:
This is probably something left-over from the reversion.  In the code that I wrote that broke things, I added a third param to the `getParameterByName` function, and you are right it was a default.  I bet that went away when Eric reverted my changes, but this line got missed.